int64 int_id
string name
float64 lat_dd
float64 lng_dd
float64 alt_rel_m
float64 safe_radius_m


